modules/serverless-metering-reports.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+// serverless-metering.adoc
+
+[id="reports-metering-serverless_{context}"]
+== Metering reports for Knative Serving
+You can run metering reports against Knative Serving by creating `Report` resources.
+Before you run a report, you must modify the input parameter within the `Report` resource to specify the start and end dates of the reporting period.
+
+[id="knative-service-cpu-usage-report_{context}"]
+== knative-service-cpu-usage-report
+
+.YAML file
+[source, yaml]
+----
+apiVersion: metering.openshift.io/v1alpha1
+kind: Report
+metadata:
+  name: knative-service-cpu-usage
+spec:
+  reportingStart: '2019-06-01T00:00:00Z'
+  reportingEnd: '2019-06-30T23:59:59Z'
+  query: knative-service-cpu-usage
+runImmediately: true
+----
```

Comment:
```suggestion
----
<1> Start date of the report, in ISO 8601 format
<2> End date of the report, in ISO 8601 format
<3> Either `knative-service-cpu-usage` for CPU usage report or `knative-service-memory-usage` for a memory usage report
```

Reply:
So, I would merge the call to the report to one section, and removing the second level headers (which imo should have been third level headers anyway). The only difference is explained in the `<3>` explanation.